about.html

```diff
@@ -11,6 +11,17 @@ <h1>About</h1>
 
 <h2>Staff</h2>
 
+<p>
+  Contact us via email using
+  <a href="mailto: not.kevin.888@gmail.com?subject=sq1obl.com Feedback">
```

Comment:
(p3) should this space be removed? I'm not super familiar with mailto's
```suggestion
  <a href="mailto:not.kevin.888@gmail.com?subject=sq1obl.com Feedback">
```